.travis.yml

```diff
@@ -17,15 +19,15 @@ matrix:
   allow_failures:
     - php: 7.0
 
-install:
-  - sudo apt-get update
-  - sudo apt-get -qq --fix-missing install valgrind
-
-  - wget -qO - http://www.rabbitmq.com/rabbitmq-signing-key-public.asc | sudo apt-key add -
-  - sudo add-apt-repository 'deb http://www.rabbitmq.com/debian/ testing main'
-  - sudo apt-get update
-  - sudo apt-get install --only-upgrade -y rabbitmq-server
-  - sudo service rabbitmq-server restart
+#install:
+#  - sudo apt-get update
+#  - sudo apt-get -qq --fix-missing install valgrind
+#
+#  - wget -qO - http://www.rabbitmq.com/rabbitmq-signing-key-public.asc | sudo apt-key add -
+#  - sudo add-apt-repository 'deb http://www.rabbitmq.com/debian/ testing main'
+#  - sudo apt-get update
+#  - sudo apt-get install --only-upgrade -y rabbitmq-server
+#  - sudo service rabbitmq-server restart
```

Comment:
True. The original intention is to keep it here to be able to switch to newer RabbitMQ server version during development (while Travis machines install RabbitMQ from the same source, it is often outdated), so I guess it should be kept for now.